media-sound/ncmpc/ncmpc-0.30.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit meson ninja-utils
```

Comment:
no need to inherit `ninja-utils` manually: https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/meson.eclass#n53